### PR TITLE
[TACHYON-1207] Added tests for lineage file system client

### PR DESCRIPTION
**Pull request by yupeng9**

### CODE REVIEW

File: clients/unshaded/src/test/java/tachyon/client/lineage/TachyonLineageFileSystemTest.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    Assert.assertEquals(lfs, TachyonLineageFileSystem.get());
+  }
+
+  @Test
+  public void getLineageOutStreamTest() throws Exception {
+    TachyonURI path = new TachyonURI("test");
+    Mockito.when(mLineageMasterClient.reinitializeFile("test", DEFAULT_BLOCK_SIZE, 0))
+        .thenReturn(1L);
+    OutStreamOptions options =
+        new OutStreamOptions.Builder().setBlockSizeBytes(DEFAULT_BLOCK_SIZE).setTTL(0).build();
```
Comment (aaudiber) [inline clients/unshaded/src/test/java/tachyon/client/lineage/TachyonLineageFileSystemTest.java]: should this be named something different from DEFAULT_BLOCK_SIZE? Is the purpose to check that it's passed through to reinitializedFile? If so it's purpose is to not be the default block size.

**Comment by aaudiber** [comment]
LGTM, left one minor comment

File: clients/unshaded/src/test/java/tachyon/client/lineage/TachyonLineageFileSystemTest.java (hunk 1 above)
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/lineage/TachyonLineageFileSystemTest.java]: done

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**